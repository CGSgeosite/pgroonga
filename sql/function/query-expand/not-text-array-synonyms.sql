CREATE TABLE synonyms (
  term text PRIMARY KEY,
  synonym text
);

SELECT pgroonga_query_expand('synonyms', 'term', 'synonym', 'Groonga');

DROP TABLE synonyms;
