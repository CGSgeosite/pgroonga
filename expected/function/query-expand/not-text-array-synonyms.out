CREATE TABLE synonyms (
  term text PRIMARY KEY,
  synonym text
);
SELECT pgroonga_query_expand('synonyms', 'term', 'synonym', 'Groonga');
ERROR:  pgroonga: query_expand: synonyms column isn't text[] type: <synonyms>.<synonym>
DROP TABLE synonyms;
